doc/rm/c_cpp_coding_style.md

```diff
@@ -114,6 +118,9 @@ The above rules also do not apply to system includes, which should be included b
 The first line of the comment is the summary, followed by a new line, and an optional longer description.
 Input arguments and return arguments can be documented with `@param` and `@return` if they are not self-explanatory from the name.
 
+The `@p` modifier can be used to format the following word as monospaced text in the final output.
+The documentation tool will also render markdown within descriptions.
```

Comment:
Is there a preference for `@p my_var` or `` `my_var` ``? This looks like more of a statement of fact than a style suggestion, but I'm not sure.

Reply:
I *think* `@p named_item` might auto-link to that item in the doxygen, but I would need to check. `@p` seems to only supports one word following it, I'm not sure about punctuation. Let me get back to you on this one.

Reply:
Update: The following are treated as a single-word for `@p`:
- `@p void*` (but not `@p void *`).
- `@p foo<bar>`.
- `@p foo(bar)`.
- `@p foo::bar`.

`@p named_item` does not auto-link. Use `#named_item`, `named_fn()` or `::named_item` to auto-link to declarations. This does not work for parameter names, but seems to work for most other declarations that doxygen knows about (so not `void*` either). 

I think this means we should err towards `` `foo` ``, then `#named_item`, then `@p`. I shall update the guidance.

Reply:
+1 on preferring the simpler Markdown style.

Reply:
Ok, I have removed the reference to `@p` and pointed to using backticks.